uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/option-group-mock-data.js

```diff
@@ -0,0 +1,328 @@
+      }, {
+        "id": "1110",
+        "option_group_id": "141",
+        "label": "1 1/4 Days",
+        "value": "1.25",
+        "name": "one_and_a_quarter_days",
+        "is_default": "0",
+        "weight": "6",
+        "is_optgroup": "0",
+        "is_reserved": "0",
+        "is_active": "1"
+      }, {
+        "id": "1111",
+        "option_group_id": "141",
+        "label": "1 1/2 Days",
+        "value": "1.5",
+        "name": "one_and_a_half_days",
+        "is_default": "0",
+        "weight": "7",
+        "is_optgroup": "0",
+        "is_reserved": "0",
+        "is_active": "1"
+      }, {
+        "id": "1112",
+        "option_group_id": "141",
+        "label": "1 3/4 Days",
+        "value": "1.75",
+        "name": "one_and_three_quarters_days",
+        "is_default": "0",
+        "weight": "8",
+        "is_optgroup": "0",
+        "is_reserved": "0",
+        "is_active": "1"
+      }],
+      hrleaveandabsences_leave_balance_change_type: [{
+        "id": "1099",
+        "option_group_id": "140",
+        "label": "Leave",
+        "value": "1",
+        "name": "leave",
+        "is_default": "0",
+        "weight": "0",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }, {
+        "id": "1100",
+        "option_group_id": "140",
+        "label": "Brought Forward",
+        "value": "2",
+        "name": "brought_forward",
+        "is_default": "0",
+        "weight": "1",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }, {
+        "id": "1101",
+        "option_group_id": "140",
+        "label": "Public Holiday",
+        "value": "3",
+        "name": "public_holiday",
+        "is_default": "0",
+        "weight": "2",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }, {
+        "id": "1102",
+        "option_group_id": "140",
+        "label": "Credit",
+        "value": "4",
+        "name": "credit",
+        "is_default": "0",
+        "weight": "3",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }, {
+        "id": "1103",
+        "option_group_id": "140",
+        "label": "Debit",
+        "value": "5",
+        "name": "debit",
+        "is_default": "0",
+        "weight": "4",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }, {
+        "id": "1104",
+        "option_group_id": "140",
+        "label": "Overridden",
+        "value": "6",
+        "name": "overridden",
+        "is_default": "0",
+        "weight": "5",
+        "is_optgroup": "0",
+        "is_reserved": "1",
+        "is_active": "1"
+      }]
+    },
+    /**
+     * Pick a random value out of a collection
+     *
+     * @param {array} the option group collection key
+     * @param {string} key - The sub-collection key
+     * @return {object}
+     */
+    randomValue: function (collection, key) {
```

Comment:
I'm really not sure about this. These mock data files were supposed to be pretty simple: a collection of mocked data that the tests were to use. But now we're adding these utility functions that I don't know if they should be here at all. Why don't you let the tests themselves to take care of that? If they need some random data, they can do it themselves.

Actually i see that you're using it in the mocked leave requests file. That's fine, but then don't add "noise" here, just put a private function in that file